src/Integration/Migrations/20211107215810_newMigration.cs

```diff
@@ -45,19 +46,19 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 columns: new[] { "Id", "Content", "DrugstoreId", "RecievedTime", "Response", "SentTime" },
                 values: new object[,]
                 {
-                    { 1, "Nije mi se svidela usluga", 1, new DateTime(2021, 11, 4, 22, 46, 10, 39, DateTimeKind.Local).AddTicks(109), "Nemoj da lazes!", new DateTime(2021, 11, 4, 22, 46, 10, 41, DateTimeKind.Local).AddTicks(7334) },
```

Comment:
zašto je id string? inkrement u bazi kada se dodaje novi? BIGSERIAL ukoliko je tip polja u tabeli u bazi povećava automatski id svake nove dodate n-torke